contracts_showcase/accesscontrol/package.json

```
@@ -11,10 +11,12 @@
   "type": "module",
   "license": "ISC",
   "dependencies": {
-    "@massalabs/massa-as-sdk": "^2.5.5-dev",
     "@massalabs/as-types": "^2.0.0",
+    "@massalabs/massa-as-sdk": "^2.5.5-dev",
     "@massalabs/massa-sc-deployer": "^1.3.0",
-    "@massalabs/massa-web3": "^3.0.2",
+    "@massalabs/massa-web3": "4.0.0-next.20240527122315",
```

```suggestion
    "@massalabs/massa-web3": "4.0.0-next",
```

then run npm install